cmd/healthbot/counterprobe/counterprobe.go

```diff
@@ -43,8 +40,7 @@ type CounterProbe struct {
 // New returns a *CounterProbe.
 func New(
 	chainName string,
-	endpoint string,
-	siwe string,
+	client *client.Client,
```

Comment:
Passing in a fill `*client.Client` here, makes it nicer for tests and also just been a fan of dependency injection as much as possible (vs passing in configuration for dependencies to-be-created).